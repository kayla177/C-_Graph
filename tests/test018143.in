ENTITY 277T8X66 name_277T conference
ENTITY 6X1QLN8Y name_6X1Q institution
ENTITY 2K125FZR name_2K12 author
ENTITY CACE9IYB name_CACE author
ENTITY 6ZTACV7R name_6ZTA author
ENTITY SPV2LD6N name_SPV2 author
ENTITY CNLJTIRN name_CNLJ journal
ENTITY ADPTEMHH name_ADPT journal
ENTITY VI5BXWJS name_VI5B conference
ENTITY ADY0VXY8 name_ADY0 conference
RELATIONSHIP CACE9IYB cited_by CNLJTIRN 8.2
RELATIONSHIP ADPTEMHH published_in CACE9IYB 3.3
RELATIONSHIP CNLJTIRN presented_at VI5BXWJS 9.0
RELATIONSHIP ADY0VXY8 authored_by ADPTEMHH 1.0
RELATIONSHIP 6ZTACV7R presented_at CNLJTIRN 7.9
RELATIONSHIP 277T8X66 collaborated_with ADPTEMHH 9.6
RELATIONSHIP CNLJTIRN affiliated_with SPV2LD6N 1.0
RELATIONSHIP VI5BXWJS published_in 2K125FZR 5.0
RELATIONSHIP 2K125FZR authored_by SPV2LD6N 1.2
RELATIONSHIP VI5BXWJS collaborated_with 6X1QLN8Y 2.9
RELATIONSHIP CNLJTIRN cited_by 6X1QLN8Y 4.6
RELATIONSHIP SPV2LD6N authored_by 6X1QLN8Y 2.0
RELATIONSHIP ADPTEMHH presented_at 2K125FZR 0.9
RELATIONSHIP CACE9IYB collaborated_with VI5BXWJS 4.3
RELATIONSHIP 6ZTACV7R authored_by ADY0VXY8 2.4
HIGHEST
PRINT 2K125FZR
DELETE 6X1QLN8Y
PRINT 6ZTACV7R
HIGHEST
PRINT CNLJTIRN
DELETE ADY0VXY8
DELETE 6ZTACV7R
HIGHEST
PRINT 2K125FZR
EXIT
