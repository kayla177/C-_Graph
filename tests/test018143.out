success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
277T8X66 ADY0VXY8 31.4
VI5BXWJS SPV2LD6N ADPTEMHH
success
CNLJTIRN ADY0VXY8
277T8X66 ADY0VXY8 31.4
CACE9IYB VI5BXWJS 6ZTACV7R SPV2LD6N
success
success
277T8X66 2K125FZR 22.2
VI5BXWJS SPV2LD6N ADPTEMHH
